risc0/zkvm/Cargo.toml

```diff
@@ -131,10 +131,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 features = ["client", "prove", "getrandom", "std", "unstable"]
 
 [features]
-circuit_debug = [
-  "risc0-circuit-rv32im-v2/circuit_debug",
-  "risc0-zkp/circuit_debug",
-]
+circuit_debug = ["risc0-zkp/circuit_debug"]
```

Comment:
is this repairing the `circuit_debug` feature in some way?

Reply:
I split `witgen_debug` from `circuit_debug`.